openapi/code/package.go

```diff
@@ -299,9 +297,10 @@ func (pkg *Package) definedEntity(name string, s *openapi.Schema, processedEntit
 
 func (pkg *Package) define(entity *Entity) *Entity {
 	if entity.IsEmpty {
-		if slices.Contains(pkg.emptyTypes, &entity.Named) {
-			//panic(fmt.Sprintf("%s is already defined", entity.Name))
-			return entity
+		for _, e := range pkg.emptyTypes {
```

Comment:
This is a bugfix, as the empty type identity is based on the type name, not the reference to the `Named` struct.